pinot-core/src/main/java/com/linkedin/pinot/core/realtime/converter/RealtimeSegmentRecordReader.java

```diff
@@ -55,6 +55,7 @@ public void init() throws Exception {
     columns.addAll(dataSchema.getDimensionNames());
     columns.addAll(dataSchema.getMetricNames());
     columns.add(dataSchema.getTimeFieldSpec().getOutgoingTimeColumnName());
+    columns.addAll(dataSchema.getDateTimeNames());
```

Comment:
we should deprecate usage of getTimeFieldSpec right, else we might be adding the same column twice since getDateTimeNames might return timefieldspec as well?